django_project/certification/tests/views/test_certificate_pdf.py

```diff
@@ -0,0 +1,55 @@
+# coding=utf-8
+from __future__ import absolute_import
+
+from django.test import TestCase, override_settings
+from django.test.client import Client
+from ..model_factories import UserF
+from base.tests.model_factories import ProjectF
+from ..model_factories import (
+    CertifyingOrganisationF,
+    CourseF,
+    CourseAttendeeF)
+
+
+class TestGenerateCertificate(TestCase):
+    """Tests generation of a PDF certificate with expected content."""
+
+    @override_settings(VALID_DOMAIN=['testerver', ])
+    def setUp(self):
+        """Set up before test is run."""
+
+        self.client = Client()
+        self.client.post(
+				'/set_language/', data={'language': 'en'})
+        self.user = UserF.create(**{
```

Comment:
__[Flake8]__

[E101] indentation contains mixed spaces and tabs


<!-- Comment by [SideCI](https://sideci.com) -->